SuperBuild/External_aruco.cmake

```diff
@@ -0,0 +1,79 @@
+IF(aruco_DIR)
+  FIND_PACKAGE(aruco 2.0.19 REQUIRED NO_MODULE)
+  
+  MESSAGE(STATUS "Using aruco available at: ${aruco_DIR}")
+
+  FOREACH(lib ${aruco_LIBS})
+	IF(NOT TARGET ${lib})
+		continue()
+	ENDIF()
+
+    GET_TARGET_PROPERTY(DEBUG_FILE ${lib} IMPORTED_LOCATION_DEBUG)
+    GET_TARGET_PROPERTY(RELEASE_FILE ${lib} IMPORTED_LOCATION_RELEASE)
+	
+    IF(MSVC OR ${CMAKE_GENERATOR} MATCHES "Xcode")
+      IF(EXISTS ${ITK_RELEASE_FILE})
```

Comment:
I removed the aruco files (.exe, .pdb, .dll, .lib) from that directory and rebuilt with the if(MSVC...) block removed, and for me the dll and pdb files do still get copied into PlusBuild-bin/bin/Debug.